lms/djangoapps/teams/static/teams/js/spec_helpers/team_spec_helpers.js

```diff
@@ -290,7 +265,8 @@ define([
         teamsUrl: '/api/team/v0/teams/',
         teamsDetailUrl: '/api/team/v0/teams/team_id',
         teamMembershipsUrl: '/api/team/v0/team_memberships/',
-        teamMembershipDetailUrl: '/api/team/v0/team_membership/team_id,' + testUser,
+        teamMembershipDetailUrl: '/api/team/v0/teams/',
```

Comment:
I am confused about the names here. I don't really like renaming teamMembershipDetailUrl as leaveTeamUrl, as it isn't just for leaving teams (though that may be all we are using). It's for interacting with a team membership, with the DELETE method doing deletion. I would prefer to keep the name as it was.

And then I'm confused why /api/team/v0/teams/ should be teamMembershipDetailUrl, as you are using it to get all team memberships for a given user, correct? Perhaps myTeamsUrl?